steps/s3-download/1.0.0/step.yml

```diff
@@ -0,0 +1,70 @@
+title: S3 Download
+summary: This step allows to download a file from an S3 bucket.
+description: This step allows to download a file from an S3 bucket using an Access/secret
+  keypair for authentication.
+website: https://github.com/FutureWorkshops/bitrise-step-s3-download
+source_code_url: https://github.com/FutureWorkshops/bitrise-step-s3-download
+published_at: 2020-10-13T13:41:06.598528+02:00
+source:
+  git: https://github.com/FutureWorkshops/bitrise-step-s3-download.git
+  commit: e8db70d94d97e74dfebf9e916f38020f89dc83ad
+host_os_tags:
+- osx-10.10
+- osx-10.9
+project_type_tags:
+- ios
```

Comment:
This step should not be restricted to particular kind of projects.
No project type tags should be specified.

Reply:
👍